index.html

```diff
@@ -23,6 +23,12 @@ <h3>Lorem Ipsum</h3>
       <h6>Lorem Ipsum</h6>
     </div>
   </div>
+  <div>
+    <h4>Subscribe to our newsletter</h4>
+    <input class="inputnewsletter" type="text" placeholder="Enter your e-mail">
+    <button class="btn" type="submit" name="action">Submit
```

Comment:
do you have a css class `btn`? I can't find it. If you're not using something right now, then try to not push it. This will keep your code more clean.